airflow/secrets/local_filesystem.py

```diff
@@ -0,0 +1,257 @@
+            # Ignore empty line
+            continue
+
+        if COMMENT_PATTERN.match(line):
+            # Ignore comments:
+            continue
+
+        var_parts: List[str] = line.split("=", 2)
+        if len(var_parts) != 2:
+            errors.append(
+                FileSyntaxError(
+                    line_no=line_no,
+                    message='Invalid line format. The line should contain at least one equal sign ("=").',
+                )
+            )
+            continue
+
+        key, value = var_parts
+        if not key:
+            errors.append(FileSyntaxError(line_no=line_no, message="Invalid line format. Key is empty.",))
+        secrets[key].append(value)
+    return secrets, errors
+
+
+def _parse_json_file(content: str) -> Tuple[Dict[str, Any], List[FileSyntaxError]]:
+    """
+    Parrse a file in the JSON format.
+
+    :param content:
+    :return: Tuple with mapping of key and list of values and list of syntax errors
+    """
+    if not content:
+        return {}, [FileSyntaxError(line_no=1, message="The file is empty.")]
+    try:
+        secrets = json.loads(content)
+    except JSONDecodeError as e:
+        return {}, [FileSyntaxError(line_no=int(e.lineno), message=e.msg)]
+    if not isinstance(secrets, dict):
+        return {}, [FileSyntaxError(line_no=1, message="The file should contain the object.")]
+
+    return secrets, []
+
+
+def _parse_secret_file(file_content: str, file_path: str) -> Tuple[Dict[str, Any], List[FileSyntaxError]]:
+    """
+    Based on the file extension format, selects a parser and parse the file.
+
+    :param file_content:
+    :param file_path:
+    :return:
+    """
+    if file_path.lower().endswith(".env"):
+        secrets, parse_errors = _parser_env_file(file_content)
+    elif file_path.lower().endswith(".json"):
+        secrets, parse_errors = _parse_json_file(file_content)
+    else:
+        raise AirflowException("Unsupported file format. The file must have the extension .env or .json")
+    return secrets, parse_errors
+
+
+def _read_secret_file(file_path: str) -> Dict[str, str]:
+    if not os.path.exists(file_path):
+        raise AirflowException(
+            f"File {file_path} was not found. Check the configuration of your secret backend."
+        )
+
+    log.debug("Reading file: %s", file_path)
+    with open(file_path) as f:
+        file_content = f.read()
+
+    log.debug("Parsing file: %s", file_path)
+    secrets, parse_errors = _parse_secret_file(file_content, file_path)
+    log.debug("Parsed file file: len(parse_errors)=%d, len(secrets)=%d", len(parse_errors), len(secrets))
+
+    if parse_errors:
+        raise AirflowFileParseException(
+            f"Failed to load the secret file.", file_path=file_path, parse_errors=parse_errors
+        )
+    return secrets or {}
+
+
+def _create_connection(conn_id: str, value: Any):
+    """
+    Creates a connection based on a URL or object.
+    """
+    from airflow.models import Connection
```

Comment:
If we are already importing the Connection class, why do we need `CONNECTION_PARAMETERS_NAMES`, we could get them from the class itself, we don't need hardcoding of params

Reply:
I had cyclical imports before. Maybe it somehow disappeared. I'll check it.